Assets/MixedRealityToolkit.Tools/MSBuild/Scripts/AssetScriptReferenceRetargeter.cs

```diff
@@ -374,9 +374,11 @@ private static bool IsYamlFile(string filePath)
                                 {
                                     Debug.LogError($"Encountered a MonoScript we get a null Type from: '{monoScript.name}'");
                                 }
-                                else if (type.Namespace == null || !type.Namespace.Contains("Microsoft.MixedReality.Toolkit"))
+                                else if ((type.Namespace == null) || 
```

Comment:
Why this change? We should be able to just move the script Init into Microsoft.MixedReality.Toolkit namespace?